src/SFA.DAS.CaptureTheFlag.Web/Controllers/ForumController.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DAS_Capture_The_Flag.Models.Forum;
+using DAS_Capture_The_Flag.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+
+namespace DAS_Capture_The_Flag.Controllers
+{
+    // Inject an instance of our ForumService from forum layer into the constructor
+    public class ForumController : Controller
+    {
+        private readonly IForumService _forumService;
+        private readonly IPostService _postService;
+
+        public ForumController(IForumService forumService)
+        {
+            _forumService = forumService;
+        }
+
+        public IActionResult Index()
+        {
+            IEnumerable<ForumListingModel> forums = _forumService.GetAll()
```

Comment:
Good - calling the service from the constructor means that if the logic in the service needs to change the code in the controller doesn't have to

The methods of the service and the controller could be marked as [async ](https://docs.microsoft.com/en-us/dotnet/csharp/async) which would make the app more scalable as it could run n a multi-threaded fashion